src/warehouses/components/WarehouseList/WarehouseList.tsx

```diff
@@ -142,7 +142,7 @@ const WarehouseList: React.FC<WarehouseListProps> = props => {
               <TableCell className={classes.colZones} data-test="zones">
                 {mapEdgesToItems(warehouse?.shippingZones)
                   .map(({ name }) => name)
-                  .join(", ") || <Skeleton />}
+                  .join(", ") || "-"}
```

Comment:
Skeleton should remain visible when page is loading and data is `undefined`, dash symbol should be displayed only if `mapEdtesToItems` returns `[]`.

Reply:
I'm not sure if that's applicable though? When the data is still loading, the entire list and all its fields are unpopulated:
![image](https://user-images.githubusercontent.com/35644259/121854848-f9cc4d80-ccf2-11eb-802e-e488188247e0.png)